Makefile.depends

```diff
@@ -1,43 +1,55 @@
-$(call depends,models/linkages): | .install/base/utils .install/modules/data.atmosphere .install/base/logger .install/base/remote
-$(call depends,models/lpjguess): | .install/base/utils .install/base/logger .install/base/remote
-$(call depends,models/maat): | .install/modules/data.atmosphere .install/base/logger .install/base/remote .install/base/settings .install/base/utils
-$(call depends,models/maespa): | .install/modules/data.atmosphere .install/base/logger .install/base/remote .install/base/utils
-$(call depends,models/preles): | .install/base/utils .install/base/logger .install/modules/data.atmosphere .install/base/utils
-$(call depends,models/sibcasa): | .install/base/logger .install/base/utils
-$(call depends,models/sipnet): | .install/modules/data.atmosphere .install/base/logger .install/base/remote .install/base/utils
-$(call depends,models/stics): | .install/base/settings .install/base/db .install/base/logger .install/base/utils .install/base/remote
-$(call depends,models/template): | .install/base/db .install/base/logger .install/base/utils
+$(call depends,models/ed): | .install/modules/data.atmosphere .install/modules/data.land .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/remote .install/base/settings .install/base/utils
+$(call depends,models/fates): | .install/base/utils .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/remote
+$(call depends,models/gday): | .install/base/utils .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/remote
+$(call depends,models/jules): | .install/base/utils .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/remote
+$(call depends,models/linkages): | .install/base/utils .install/modules/data.atmosphere .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/remote
+$(call depends,models/lpjguess): | .install/base/utils .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/remote
+$(call depends,models/maat): | .install/modules/data.atmosphere .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/remote .install/base/settings .install/base/utils
+$(call depends,models/maespa): | .install/modules/data.atmosphere .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/remote .install/base/utils
+$(call depends,models/preles): | .install/base/utils .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/modules/data.atmosphere .install/base/utils
+$(call depends,models/sibcasa): | .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/utils
+$(call depends,models/sipnet): | .install/modules/data.atmosphere .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/remote .install/base/utils
+$(call depends,models/stics): | .install/base/settings .install/base/PEcAn.DB.Rcheck/PEcAn.DB .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/utils .install/base/remote
+$(call depends,models/template): | .install/base/PEcAn.DB.Rcheck/PEcAn.DB .install/base/PEcAn.logger.Rcheck/PEcAn.logger .install/base/utils
```

Comment:
All these added lines that contain `PEcAn.*.Rcheck` are in error. This is arguably a bug in the `generate_dependencies.R` script (It looks for everything that looks like an R package anywhere in the PEcAn directory, even inside Rcheck folders), but the easiest fix will be to remove all the `*.Rcheck` directories from your PEcAn folder and run `generate_dependencies.R` again.

Reply:
Thanks @infotroph, I will do the changes ASAP